devtools-integration-tests/testerina-tests/config-api-test/tests/config-test.bal

```diff
@@ -0,0 +1,27 @@
+// Copyright (c) 2020 WSO2 Inc. (http://www.wso2.org) All Rights Reserved.
+//
+// WSO2 Inc. licenses this file to you under the Apache License,
+// Version 2.0 (the "License"); you may not use this file except
+// in compliance with the License.
+// You may obtain a copy of the License at
+//
+// http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing,
+// software distributed under the License is distributed on an
+// "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+// KIND, either express or implied.  See the License for the
+// specific language governing permissions and limitations
+// under the License.
+
+import ballerina/test;
+import ballerina/config;
+// This tests the functionality of reading config
+
+@test:Config{
+}
+function test1 () {
```

Comment:
IMO it would be better if we can move this test to https://github.com/ballerina-platform/ballerina-distribution/tree/master/stdlib-integration-tests since this is more of a test for config module right?WDYT?

Reply:
Moved config to `stdlib-integration-test`